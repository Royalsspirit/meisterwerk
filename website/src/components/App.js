import React from 'react'
import logo from './logo.svg'
import './App.css'
import List from './List.container'
import Container from 'react-bootstrap/Container'
const App = () => {
    return (
        <div className="App">
            <header className="App-header">
                <img src={logo} className="App-logo" alt="logo" />
            </header>
            <Container fluid>
                <List />
            </Container>
        </div>
    )
}

export default App
