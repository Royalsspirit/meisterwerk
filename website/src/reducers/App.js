const intialState = {
	commits: [],
	list: [],
}
const repositories = (state = intialState, action) => {
	console.log('action',action)
  switch (action.type) {
    case 'LIST_REPO':
      return {
        ...state,
	list: action.data,
      }
    case 'LIST_REPO_FAIL':
      return {
	...state,
	errorMessage: action.errorMessage
      }
    case 'LIST_COMMITS':
      return {
	...state,
	commits: action.data
      }
    case 'LIST_COMMITS_FAIL':
      return {
	...state,
	errorMessage: action.errorMessage
      }

    default:
      return state
  }
}

export default repositories
