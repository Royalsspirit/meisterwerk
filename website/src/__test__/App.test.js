import React from 'react';
import App from '../components/App';
import { shallow } from 'enzyme';

describe('App component', () => {
  const getWrapper = shallow(<App />);

  it('render without crashing', () => {
    expect(getWrapper).toMatchSnapshot();
  });
})