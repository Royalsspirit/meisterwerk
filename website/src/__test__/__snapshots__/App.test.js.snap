// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component render without crashing 1`] = `ShallowWrapper {}`;

exports[`test app render without crashing 1`] = `ShallowWrapper {}`;
